File: warmup/python2/manasa-and-stones.py
for _ in xrange(int(raw_input())):
    n = int(raw_input()) - 1
    a1 = int(raw_input())
    b1 = int(raw_input())
    a, b = min(a1, b1), max(a1, b1)
    current = a * n
    maximum = b * n
    diff = b - a
    if a == b:
        print current
    else:
        x = []
        while(current <= maximum):
            x.append(current)
            current += diff
        print " ".join(map(str, x))

File: warmup/java/Manasa_and_Stones.java
import java.util.*;

public class Manasa_and_Stones {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int T = sc.nextInt();

        while(T>0) {
            int n = sc.nextInt();
            n = n - 1;
            int a1 = sc.nextInt();
            int b1 = sc.nextInt();
            int a = Math.min(a1, b1);
            int b = Math.max(a1, b1);

            int current = a * n;
            int maximum = b * n;
            int diff = b - a;

            if (a==b)
                System.out.println(current);
            else{
                String al = "";
                while (current <= maximum) {
                    al+=(int)current + " ";
                    current += diff;
                }
                    System.out.println(al);
            }
            T--;
        }
        sc.close();
    }
}